nekoyume/Assets/_Scripts/UI/Tween/AnchoredPositionSingleTweener.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using DG.Tweening;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Nekoyume.UI.Tween
+{
+    // todo: `AnchoredPositionSingleTweener`로 바꾸고, 좌표계를 선택할 수 있도록. `RotateSingleTweener` 참고.
+    [RequireComponent(typeof(RectTransform))]
+    public class AnchoredPositionSingleTweener : DOTweenBase
+    {
+        public enum Single
+        {
+            X,
+            Y,
+            Z
+        }
+
+        [SerializeField]
+        private float endValue = 0f;
+
+        [SerializeField]
+        private bool snapping = false;
+
+        [SerializeField]
+        private bool isFrom = false;
+
+        public Single single = Single.Z;
+
+        private RectTransform _rectTransformCache;
+        private Vector2? _originAnchoredPositionCache;
+        private Vector2 _startPosition;
+        private Vector2 _endPosition;
+
+        public RectTransform RectTransform => _rectTransformCache
+            ? _rectTransformCache
+            : _rectTransformCache = GetComponent<RectTransform>();
+
+        private Vector2 OriginAnchoredPosition =>
+            _originAnchoredPositionCache
+            ?? (_originAnchoredPositionCache = RectTransform.anchoredPosition).Value;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            Assert.NotNull(RectTransform);
+            Assert.AreEqual(OriginAnchoredPosition, _originAnchoredPositionCache);
+        }
+
+        private void OnEnable()
+        {
+            _startPosition = OriginAnchoredPosition;
+            _endPosition = OriginAnchoredPosition;
+
+            switch (single)
```

Comment:
Is it ok to not covering all cases?

Reply:
Yes. anyway, RectTransform.AnchoredPosition is `Vector2`.
I decided that we would not have to worry about the z-axis.

Reply:
There are only x and y in [RectTransform.anchoredPosition](https://docs.unity3d.com/ScriptReference/RectTransform-anchoredPosition.html) because it's type is [Vector2](https://docs.unity3d.com/ScriptReference/Vector2.html)
So the `AxisType.Z` should removed.